### PR TITLE
feat: select peer to dial from drop down

### PR DESCRIPTION
**Pull request by adklempner**

### CODE REVIEW

File: examples/light-js/index.html [hunk 1]
```diff
@@ -87,6 +108,37 @@
 
       statusDiv.innerHTML = "<p>Starting Waku node.</p>";
       await node.start();
+
+      // Queries all peers from store and updates list of connected peers
+      const updatePeersList = async () => {
+        // Generate <p> element with connection string from each peer
+        const peers = await node.store.peers();
```
Comment (weboko) [inline examples/light-js/index.html:115]: `node.store` refers to `Store` protocol and though it seems to be possible to get peers from there we should try using `connectionManager` or `libp2p` directly.

image

File: examples/light-js/index.html [hunk 2]
```diff
@@ -10,18 +10,37 @@
...
+      <button id="getPeersButton" type="button">Refresh Peers</button>
+      <select id="peer-select">
+        <option value=""></option>
+      </select>
+    </div>
+    <div>
+      <label for="remote-multiaddr">Remote peer's multiaddr</label>
+    </div>
+    <div>
+      <input id="remote-multiaddr" type="text" value="" style="width: 100%" />
```
Comment (weboko) [inline examples/light-js/index.html:36]: nit: let's add some styling - this field is soooo long :)

File: examples/light-js/index.html (hunk 1 above)
Comment (adklempner) [reply examples/light-js/index.html]: Updated to use `node.libp2p.peerStore`

File: examples/light-js/index.html (hunk 2 above)
Comment (adklempner) [reply examples/light-js/index.html]: Added a right margin to parent div and made the maximum width for the input 900px; this is about as wide it needs to be to display the entire multiaddr

**Review by weboko — APPROVED** [review]
Congrats with first PR 🎉

**Comment by fryorcraken** [comment]
Is that related to a specific issue?

**Comment by adklempner** [comment]
it checks off 3 of the boxes listed as requirements for this issue: https://github.com/waku-org/js-waku-examples/issues/248#issuecomment-1783628179

**Comment by fryorcraken** [comment]
> it checks off 3 of the boxes listed as requirements for this issue: [#248 (comment)](https://github.com/waku-org/js-waku-examples/issues/248#issuecomment-1783628179)

Ah great. I recommend to mention the issue in the description of PRs.

**Pull request merged**